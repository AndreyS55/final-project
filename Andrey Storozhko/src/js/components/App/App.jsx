import React from 'react';
//import { BrowserRouter } from "react-router-dom";

//import styles from './App.scss';

const App = () => <h1>Hello</h1>;

export default App;