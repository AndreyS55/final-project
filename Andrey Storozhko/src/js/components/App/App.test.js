import React from 'react';
import App from './App';

it('should render a App', () => {
  const wrapper = shallow(
    <App>Hello</App>
  );
  expect(wrapper).toMatchSnapshot();
});
