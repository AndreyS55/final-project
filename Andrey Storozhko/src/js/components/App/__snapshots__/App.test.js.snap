// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a App 1`] = `
<h1>
  Hello
</h1>
`;
